submissions/xT0PERx/js-dom/code.js

```diff
@@ -0,0 +1,65 @@
pn3zQwJEzaI/ZqUAIo4dGKQogJhoYHTUnwFCKtV4u00U7waYo9GRzrU4XhTO3TJFUzYJI2hUgIoUkPDo8EYCDe2papZxYQKIswApQRQpIaGQy0gHG/ujUGvlbZp0nxNot9GHDrR9BgIN7jrvVHFhA5pIJxoakgNjYskTLghO+OpjVup0iF8JzHR1CRMbydq0NV1cI+L3499KbmKCQeEo36iqSE0NAZqA6GaMFdbjJg2JnR6ZNmMRFNDGDo1OvirZuMiE8FRqR4h6fbLCV5NqUxkaAykYEJps+/PawombBBWYkKuJnGFQ7yqyPsyXYGzFbGqtDEhi4hC7IQAisyj0ZCDCaVNf4HCdtpAOEBElZi0OyiRoZFQBRNucqiYSsF1OKCJKjEhV0NkaEzkYMLK5jeNFR1M2EREQVR8roYoR1NFEibMs16CVI6DCWkicxrr1RAZGg01KkBY2cZ3buQGJmSJemKsV0OUo2kjCddhm/Ru84jx6pCtk6ROY70aapL9AgZqYMLKdjYHEAaYkEbd7EViTQ1J6KRAFQJhdTv7ATENj7BL8u2SmMnuUim7R2MjpwqE0mYdtqXUJiask1Ri3BsIBKHToFPFhNVtbXjEsE1M2EXZp/BiTA2BRyNCFQKhtL3dgFR2Spg9Y/MYQ5jdo0HdNiZUtrcrF9NkMWEne9otJoDKbmhU5GDC8O54G5VKe4S97LnTiKnJnqMxEd3GhFusQvyBHnpKGLOEP16RGajs09stpGDCrVYhjIn1KWHmYT8SQGXO0fQR6xG2t7tzHMPWMWEPhV8XSlLEq8lsaMDJx4SRLY03La47JcyaHo6Ymqyhk4yaU8Jtb2/Is90pYcZJYT6cq8kYOvGo62BCZfsfz7R6M8KMxiaUq8k6vV1BjSlh/jvNK77RmRJmTJ6GTE1Gj8ZArEfo7OJ7MGLPI+yhbIv4Q7majIaG7TUwoVLwIraMaiOPsJdtodRdcLI7m6ERUNMjbOzmE6/ajLCXLaERMDXZXkEwUX0wAEJn0znSJLWQR9hBdJZ2Ggigsnk0DdT0CBu72iaW6SCPsJupnQZMTSZDwyHWI2zsbg9VbkbYTd/TbaqAV0NNVhcwUbeJCRvkK80KEz+YEtZ73dXtNPBiQpbQqYlYrw6bu/xymD0jpFGGRUo+Y5oldFIR7REOdrsLrjQjpDMEGT5TkyFH00dddkq428/bMTDgY0K2vjqj4cvVrA6d+G6P9uqwueutmrkZYXO1f+rL1az2aCqo7hE2d2hmZmqgKWFzZebNF0Ct9Ggs1J0S7sF+4sasDgerp74XLyaUXq0wNCbq1D1CdvPThauloimhQ6/qiosAatWsE19H9SnhzryZgJodj7CxconGwqtZZWgkDIgJI9+b2o0MNCN0VnhvC69mhaHhPEAgpPdlQ3i1MyVUnPTJmsUMVLqhsVGXntXhvnx6gm/Up4TVQer0Pj+f7E4NncDK0FPC7n60USx7RtiusqnWZjbZnZqj4VlETwm7+2BH5xLqM0IpdVXfbC/B1NCpjdgpYXewL23Uk8LOCCsoZWHdfMOZSfIpOvjbHmG3u/ux3i9zQSilvHMy2zUoJXSScUDhEfaKfr9wXVnsjLBVTVlg63k1JSrRo3FRvTklzPGC1abVGswI9Xay++YFUMmhE0RMAOgR0vv3SXDemROqSuKY4QVQiYYGxonmlLDe3X4Sf7X6zRmhria+deKZGmoSf5BBveaMsLOr9GG65MacUGAT5mu8XE2CoeFpNJgR1vevE06lK3NCLmnkxx8QPYvdbhVG+sGckN2LiCJGfHVByNXrsXcJXk1CMriB8FKrprcGYpfJtXSZzoxQ4OROM84lwR9YPY4r6mDAKWF915mZNNnKglDuxEUGj1R86KSgRmNGWN+XkCleVnVOaFn1mDTxkIoNnRzUcGaE9FYWWK4htTInFMV6jMW4Lsd4NBhwRrgneYs04SF/Sui6XTpyu9+paOjUQBA8zwl3M1NIIr66qEPbjY6LHyI5GhgmFMWZEbL7E/Qmy5QWhLYb8W5uwobGrPeUBSG93x8Im0uTFnXoukrIDR+FDI2Guu32nJDd53HCL6PCzQlFsRpaq3EWPBXV2wtCdu3dx7cmt7UkFFvBdH+gG1qIrS4Im/s9EAYl6ktCcRyYXPQ3UgE1qgvCwTZeFilOlrogtEQBObFJJQkp0oKwIe1V4mm1ZGFBaFlyPSaaYliEXyueETa29KJBgeI4d0FoyU5kbZiB6Iq0JHx+gFCL3LIOZUsKzWlwyKksCZ8lIPRFQJwTyrKKmksXjm+jamtJ6Dy3PjiXxS0JOZlrLloqjIItAJwTOpVnCgiDhuAuCEESmtbVGCm6j9B5TuNgWK7qJ+TG9Y59YNC9CgAuCJ19moAhl6FjxBmhIAhV1IQK1H2Ezr5l70ll6pafUNAHLdVP6Gx7CX7xYnTZR6iCdB/hDhaoFy9+zFl+wrGPUNmvGbTc4tR4wurW3ifcuFzdQwwRVp+3EQ2q34rWYfu5BPTZxOhCiFDZ39R9Tsm6n7CyoT3ldiqjtSTc4hu92xSj6zPC6v9dC51LlDBiS/8/bKFzmS19LP0/uDEpkvX9W2Txohe96EVr63/iN69zrIQJkgAAAABJRU5ErkJggg==><p> Tesla (ранее Tesla Motors) — американская компания, производитель электромобилей и (через свой филиал SolarCity) решений для хранения электрической энергии.</p><p>Tesla разворачивает сеть «Суперзарядок» (англ. Supercharger) — станций для зарядки электромобилей, разработанных для того, чтобы на автомобилях Tesla можно было совершать длительные поездки. </p>',
```

Comment:
Avoid using images in base64 for no reason.

Reply:
Fixed